django_scaffold/settings.py

```diff
@@ -17,10 +18,21 @@
 # Application definition
 
 INSTALLED_APPS = [
-    "django_scaffold",  # must be first to override migrate command
```

Comment:
No longer needed as legacy_migrations has the `migrate` command we want to use now